jetbrains-core/tst-213/software.aws.toolkits.jetbrains/core/explorer/AwsExplorerNodeProcessorTest.kt

```diff
@@ -0,0 +1,112 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.core.explorer
```

Comment:
folder is wrong for kotlin files:
```
.../software.aws.toolkits.jetbrains/...
```

vs

```
.../software/aws/toolkits/jetbrains/...
```

there are several other files with the same problem

Reply:
moved the files to the correct path